### PR TITLE
chore: turn v1 reporting off

### PR DESCRIPTION
**Pull request by deepak1725**
Turning V1 Venus Reporting OFF

- Move Sentry Reporting  out to the main function
- Move Ingestion Reporting out to the main function.

Signed-off-by: Deepak Sharma <deepshar@redhat.com>

### CODE REVIEW

**Comment by prashbnair** [comment]
@deepak1725 are you adding new functionality here. To move something code needs to be deleted from one place and added to another, I only see code being added here. Only the V1 code seems to be removed.

**Comment by deepak1725** [comment]
> @deepak1725 are you adding new functionality here. To move something code needs to be deleted from one place and added to another, I only see code being added here. Only the V1 code seems to be removed.

No No, Actually, Sentry Reporting and Ingestion Reporting were lying along with v1 Venus Reporting, If we remove that code directly, then both of these will also not get triggered, So that why I add few lines in the main function to execute them sequentially along with v2 venus reporting.

File: f8a_report/helpers/ingestion_helper.py [hunk 1]
```diff
@@ -50,3 +54,26 @@ def ingest_epv(missing_latest_nodes):
         logger.error("Error while ingesting missing versions {}".format(e))
 
     return result
+
+
+def generate_ingestion_report(start_date: str, end_date: str, frequency='daily') -> Dict:
```
Comment (deepak1725) [inline f8a_report/helpers/ingestion_helper.py:59]: @prashbnair This function acts as an Entry Point to the Ingestion reporting. 
I rewrote this just to match v2 standards of moving RDS query functions to `db_gateway`

File: f8a_report/helpers/sentry_report_helper.py [hunk 2]
```diff
@@ -138,3 +138,11 @@ def retrieve_events(self, issue_id):
         except NameError as e:
             logger.error('Name not found while parsing. Reason: %r' % e)
         return events
+
+
+def generate_sentry_report(start_date: str, end_date: str):
```
Comment (deepak1725) [inline f8a_report/helpers/sentry_report_helper.py:143]: This is Entry Point for Sentry Reporting.

Comment (yzainee) [reply f8a_report/helpers/sentry_report_helper.py:143]: What about the logic of collecting the sentry error report which is currently in the code? where was it getting called from earlier?

Comment (deepak1725) [reply f8a_report/helpers/sentry_report_helper.py:143]: @yzainee here, along with v1 reporting https://github.com/fabric8-analytics/f8a-stacks-report/blob/master/f8a_report/helpers/report_helper.py#L868

Comment (yzainee) [reply f8a_report/helpers/sentry_report_helper.py:143]: strange logic existed.. im fine with the changes now.

**Review by yzainee — APPROVED** [review]

**Pull request merged**